src/bin/stratisd.rs

```diff
@@ -363,7 +363,6 @@ fn run(matches: &ArgMatches, buff_log: &buff_log::Handle<env_logger::Logger>) ->
     */
     const FD_INDEX_UDEV: usize = 0;
     const FD_INDEX_SIGNALFD: usize = 1;
-    const FD_INDEX_DUMP_TIMERFD: usize = 2;
```

Comment:
Mind the gap! You're now missing a value in the order, and that's probably going to cause trouble. Pay close attention to how the indices for ```fds``` are managed.

Reply:
I also need to update the comments above.  Hopefully that should be done now.